BUILDING.md

```diff
@@ -1,244 +1,102 @@
-If you're using an EC2 instance, the [EC2 instance's IAM role](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) will be used automatically if available.
-
-For a simple start, pick an [EC2 region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions), then run:
-
-```shell
-cargo make -e PUBLISH_REGIONS=your-region-here ami
+### Setup Twoliter.toml for testing
+To consume a private copy of the bottlerocket core kit with your changes built into it, you need to define the vendor that points to your container registry in `Twoliter.toml` and adjust the core kit dependency:
 ```
-
-Note that the task ("ami") must come **after** the arguments to `cargo make` that are specified with `-e`.
-
-Your new AMI ID will be printed after it's registered.
-
-If you built your image for a different architecture or variant, just use the same arguments here:
-
-```shell
-cargo make -e PUBLISH_REGIONS=your-region-here -e BUILDSYS_VARIANT=my-variant-here ami
+[vendor.my-vendor]
+registry = "####.dkr.ecr.us-west-2.amazonaws.com"
+[[kit]]
+name = "bottlerocket-core-kit" # Name of your ECR repo
+version = "2.x.y" # your version tag you want to test
+vendor = "my-vendor"
 ```
-
-(There's a lot more detail on building and managing AMIs in the [PUBLISHING](PUBLISHING.md) guide.)
-
-## Use your image
-
-See any of the setup guides tailored to the various execution environments for information on running Bottlerocket images:
-
-* [Setup guide for Kubernetes](QUICKSTART-EKS.md)
-* [Setup guide for Amazon ECS](QUICKSTART-ECS.md)
-* [Setup guide for VMware](QUICKSTART-VMWARE.md)
-* [Setup guide for QEMU/KVM](QUICKSTART-LOCAL.md)
-
-## Publish your image
-
-See the [PUBLISHING](PUBLISHING.md) guide for information on deploying Bottlerocket images and repositories.
-
-## Building out-of-tree kernel modules
-
-To further extend Bottlerocket, you may want to build extra kernel modules.
-The specifics of building an out-of-tree module will vary by project, but the first step is to download the "kmod kit" that contains the kernel headers and toolchain you'll need to use.
-
-### Downloading the kmod kit
-
-kmod kits are included in the official Bottlerocket repos starting with Bottlerocket v1.0.6.
-Let's say you want to download the kit for building x86_64 modules for v1.11.0 and variant aws-k8s-1.24.
-
-First, you need tuftool:
-```shell
-cargo install tuftool
+Any time you change the vendor and/or version of the kit above you need to run twoliter update to update the Twoliter.lock
 ```
-
-Next, you need the Bottlerocket root role, which is used by tuftool to verify the kmod kit.
-This will download and verify the root role itself:
-```shell
-curl -O "https://cache.bottlerocket.aws/root.json"
-sha512sum -c <<<"a3c58bc73999264f6f28f3ed9bfcb325a5be943a782852c7d53e803881968e0a4698bd54c2f125493f4669610a9da83a1787eb58a8303b2ee488fa2a3f7d802f  root.json"
+twoliter update
```

Comment:
Need to check if `twoliter` is exposed in the `PATH` in the environment. 

Otherwise, it is consumable in `./tools/twoliter/twoliter update`.